recipes/mdsplus/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "mdsplus" %}
+{% set version = "stable_release-7-7-9" %}
+{% set sha256 = "5b54e4f7bb00afee7ca9ad0fd7f5fda69afe39c817f68c7739939b1415f05008
+" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version | replace('-','.') }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MDSplus/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch.osx  # [osx]
+    - configure.patch.linux  # [linux]
+    - mdsdcl_show_version.c.patch
+    - Makefile.in.patch
+    - mdsdcl_Makefile.in.patch
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - libxml2
+    - freetds
+    - readline
+    - ncurses
+    - unixodbc
+    - pyodbc
+    - gcc
+    - python 2.*
+    - xorg-libx11
+    - xorg-libxt
+    - openmotif
+    - openmotif-dev
+    - toolchain
+    - perl
+    - setuptools
+  run:
+    - libxml2
+    - freetds
+    - readline
+    - ncurses
+    - unixodbc
+    - pyodbc
+    - libiconv
+    - xorg-libx11
+    - xorg-libxt
+    - openmotif
+    - openmotif-dev
+    - python 2.*
+    - numpy
```

Comment:
I think that these need appropriate pinnings. See https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml for values

Reply:
I thought that numpy only needed to be pinned if it was used in the build process, which here it is not.

Reply:
yeah, you are right about numpy, but I think ncurses and some others need to be pinned even as run deps.